swo/mpt/extensions/runtime/djapp/conf/default.py

```diff
@@ -132,7 +132,8 @@
             "style": "{",
         },
         "rich": {
```

Comment:
IMO when the extension is running NOT in local/dev environment, the rich logging handler must be disabled.
Probably is better to add the OTEL fields to the verbose formatter.

Reply:
added to both, thanks